### ISSUE TITLE
[Bug]:  Unable to open the terminal and preview pictures

### ISSUE THREAD

**Issue by nulllt** [coder/code-server#5956]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS/Web Information

- Web Browser: edge
- Local OS: Windows
- Remote OS: Ubuntu
- Remote Architecture:x86
- `code-server --version`:4.9.1

### Steps to Reproduce

1. 
2.
3. sudo ./code-server

click terminal button
click a image file

### Expected

terminal opened
The image is shown

### Actual

Terminal is blank, nothing is here
No image shown

### Logs

Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'sha256-/9/YQU12wvTeVXCsIGB4shLwdWrMceCpKojfkloNjPU=' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-KY8PSlaoKyiU7KHD4VCtZF1UHpfDT01zYDBFFMjeJyk='), or a nonce ('nonce-...') is required to enable inline execution.

/http-9997/77726476706e69737468656265737421a1a013d2766726012a58c7fecd07/stable-6261075646f055b99068d3688932416f2346dd3b/static/out/vs/workbench/contrib/webview/browser/pre/index.html?id=e2f4799e-a101-4f47-8f21-6a5b4812c085&origin=c43e7f9e-6c55-4001-8aec-4f6c593466f2&swVersion=4&extensionId=vscode.media-preview&platform=browser&vscode-resource-base-authority=vscode-resource.vscode-cdn.net&parentOrigin=http%3A%2F%2F172.17.140.252%3A9997&remoteAuthority=172.17.140.252%3A9997:30 

VM577:609 TRACE [History stack global-default]: index: 4, navigating: false
- group: 0, editor: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/mask.png, selection: <none>
- group: 0, editor: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/performance.png, selection: <none>
- group: 0, editor: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/tennis_panoptic.png, selection: <none>
- group: 0, editor: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/tennis_semantic.jpg, selection: <none>
- group: 0, editor: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/mask.png, selection: <none>
			
VM577:609 TRACE onWillActivateByEvent:  onCustomEditor:imagePreview.previewEditor
VM577:609 TRACE MainThreadFileSystem#$unwatch(): request to stop watching (session: 0.6545383926104886)
VM577:609 TRACE MainThreadFileSystem#$watch(): request to start watching (extension: vscode.media-preview, path: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/tennis_panoptic.png, recursive: false, session: 0.7393912533506555)
VM577:609 DEBUG Webview(e4ad66c2-f2d3-4c3c-acc1-09eb93921cc3): will update content
VM577:609 TRACE [History stack global-default]: notifyNavigation() (editor: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/tennis_panoptic.png, event: <none>)
VM577:609 TRACE [History stack global-default]: notifyNavigation() not ignoring (editor: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/tennis_panoptic.png, event: <none>)
VM577:609 TRACE [History stack global-default]: onNonSelectionAwareEditorNavigation() (editor: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/tennis_panoptic.png, event: <none>)
VM577:609 TRACE [History stack global-default]: add() (editor: vscode-remote://172.17.140.252:9997/home/user/model/yolov7/figure/tennis_panoptic.png, event: <none>)

### Screenshot/Video

![QQ截图20230109221536](https://user-images.githubusercontent.com/114355008/211330975-9db6394b-93cd-4eac-b4b5-212443fe8da1.png)
![QQ截图20230109221707](https://user-images.githubusercontent.com/114355008/211331006-26a26363-f2b3-400a-823e-2cbba2ea70a6.png)

### Does this issue happen in VS Code or GitHub Codespaces?

- [X] I cannot reproduce this in VS Code.
- [X] I cannot reproduce this in GitHub Codespaces.

### Are you accessing code-server over HTTPS?

- [X] I am using HTTPS.

**Comment by jsjoeio** [comment]
Can you try it in an incognito window and tell me if it still happens?

**Comment by jsjoeio** [comment]
Hmm... I''m using 4.9.1 and do not have this issue. 
<img width="1712" alt="image" src="https://user-images.githubusercontent.com/3806031/211372281-1bade825-386b-4ac4-a37c-ac4a65db6588.png">

Are you using HTTPS? 

And how did you install code-server?

**Comment by nulllt** [comment]
I used the release version of code-server-4.9.1-linux-amd64.tar.gz, and directly ran the code-server file after decompression.
As for https, it may also be a problem with the webvpn system. I access the local website through the webvpn system. So this may not be a problem with [vscode[](url)](url)
I have also built the git service on the intranet server and accessed it outside the campus through the webvpn system. It works well
![QQ截图20230110015330](https://user-images.githubusercontent.com/114355008/211375477-b45c645b-cd28-4e45-9569-74ba54f8eb7d.png)
![微信图片_20230110015601](https://user-images.githubusercontent.com/114355008/211375492-30a831de-547f-4b30-852d-8337fb3ad93e.png)
